### PR TITLE
HOTT-1545: Update the ReferencedSearchReference

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/HOTT-1545

### What?
Add productline_suffix field to get ReferencedSearchReference endpoint

### CODE REVIEW

File: source/v2/openapi.yaml [hunk 1]
```diff
@@ -1164,9 +1164,10 @@ paths:
       parameters:
         - name: query[letter]
           in: query
-          required: true
+          required: false
```
Comment (alexdesi) [inline source/v2/openapi.yaml:1167]: now, it is optional.

**Comment by alexdesi** [comment]
Screenshot 2022-05-19 at 14 37 21
Screenshot 2022-05-19 at 14 33 00
Screenshot 2022-05-19 at 14 28 18

File: source/v2/openapi.yaml [hunk 2]
```diff
@@ -1164,9 +1164,10 @@ paths:
...
           description: |
             Retrieve search references started with  `query[letter]` letter.
+            If param letter is not provided, this endpoint responses with the list of all the SearchReferences.
```
Comment (willfish) [inline source/v2/openapi.yaml]: This reads a bit strangely :thinking: 

```suggestion
            Without a query parameter all search references are returned in the response.
```

Comment (alexdesi) [reply source/v2/openapi.yaml]: 👍

**Review by willfish — APPROVED** [review]

**Pull request merged**